From: Philipp Zabel <p.zabel@pengutronix.de>
Date: Tue, 16 May 2023 10:04:03 +0200
Subject: [PATCH] Revert "cmake: Fix find_* support when cross compiling"

PTXdist uses CMAKE_FIND_ROOT_PATH [1] to instruct CMake where to find
package information when cross-compiling. Stop overwriting this with
the contents of CMAKE_PREFIX_PATH [2] in the build system.

[1] https://cmake.org/cmake/help/v3.25/variable/CMAKE_FIND_ROOT_PATH.html
[2] https://cmake.org/cmake/help/v3.25/variable/CMAKE_PREFIX_PATH.html

This reverts commit b7c5ad2eed4405140fd062e5b915950011f4d7d0.

Signed-off-by: Philipp Zabel <p.zabel@pengutronix.de>
---
 scripts/CMakeLists.txt | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/scripts/CMakeLists.txt b/scripts/CMakeLists.txt
index 94c8528c83a4..b8aebdd6eca6 100644
--- a/scripts/CMakeLists.txt
+++ b/scripts/CMakeLists.txt
@@ -123,8 +123,4 @@ if (MIMALLOC_INSTALL_DIR)
     list(APPEND CMAKE_PREFIX_PATH ${MIMALLOC_INSTALL_DIR})
 endif()
 
-if (CMAKE_CROSSCOMPILING)
-    set(CMAKE_FIND_ROOT_PATH ${CMAKE_PREFIX_PATH} PARENT_SCOPE)
-else()
-    set(CMAKE_PREFIX_PATH ${CMAKE_PREFIX_PATH} PARENT_SCOPE)
-endif()
+set(CMAKE_PREFIX_PATH ${CMAKE_PREFIX_PATH} PARENT_SCOPE)
